low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -455,9 +455,8 @@ def _generate_host_update(self):
                 self.high_usage['memory'] = True
 
                 # Create the memory data message and hand it over to the egress processor to transmit
-                fault_event = "Host memory usage increased to %s, beyond configured threshold of %s for more than %s seconds" \
-                            %(self._node_sensor.total_memory["percent"], self._host_memory_usage_threshold,
-                                self._high_memory_usage_wait_threshold)
+                fault_event = "Host memory usage has increased to {}%, beyond the configured threshold of {}% for more than {} seconds.".format(
+                    self._node_sensor.total_memory["percent"], self._host_memory_usage_threshold, self._high_memory_usage_wait_threshold)

```

Comment:
Now that we are touching the lines should we keep the 80 char limit?

Reply:
Yes, but seems like I can reduce it only if I break the messages in parts.

Reply:
Something like this, but I remember there were guidelines for not using f-strings for error/fault messages due to translation issues. 

```suggestion
               fault_event = f"Host memory usage has increased to {self._node_sensor.total_memory['percent']}%," + \
                    f"beyond the configured threshold of {self._host_memory_usage_threshold}%" + \
                    f"for more than {self._high_memory_usage_wait_threshold} seconds."

```

Reply:
we can break it like this, but it takes a lot of lines for sure.
```python3
        fault_event = "Host memory usage has increased to {}%" \
                      ", beyond the configured threshold of {}%" \
                      " for more than {} seconds.".format(
                            self._node_sensor.total_memory["percent"],
                            self._host_memory_usage_threshold,
                            self._high_memory_usage_wait_threshold
                        )
```

Reply:
@sumedhak27 looks better.
@cdeshmukh @thavanathan please give your suggestions.

Reply:
Agree

Reply:
Done